autogluon/utils/tabular/metrics/classification_metrics.py

```diff
@@ -254,3 +257,49 @@ def prior_log_loss(frac_pos, task):
     score = (pac - base_pac) / np.maximum(eps, (1 - base_pac))
 
     return score
+
+
+def confusion_matrix(solution, prediction, labels=None, weights=None):
```

Comment:
Regarding the 'normalize' argument, I would highly recommend we add this, as most of the time this is more important than the raw numbers for analytics. This should live in this function and be applied after the standard confusion matrix is constructed.